beancount_n26/__init__.py

```diff
@@ -138,6 +144,19 @@ def __init__(
 
         self._translation_strings = _translation_strings_for(self.language)
 
+        for account, payees in self.account_to_payees.items():
+            for payee in payees:
+                if payee in self.payee_to_account:
+                    logging.warning(f"{payee} in multiple accounts")
```

Comment:
Should we raise an error here instead?

Reply:
Didn't really know what to raise here so I just raised an AssertionError